### PR TITLE
New Adapter: Groupm (Alias Of PubMatic)

### PR DESCRIPTION
**Pull request by pm-isha-bharti**

### CODE REVIEW

**Comment by mansinahar** [comment]
Hey @pm-isha-bharti! Just wanted to check in on the progress on this one in regards to the above comments.

**Comment by pm-isha-bharti** [comment]
> Hey @pm-isha-bharti! Just wanted to check in on the progress on this one in regards to the above comments.

Hi @mansinahar I have created this PR: https://github.com/prebid/prebid.github.io/pull/3399

**Review by bsardo — APPROVED** [review]

File: static/bidder-params/groupm.json [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+			"description": "An ID which identifies the ad slot"
+		},
+		"pmzoneid": {
+			"type": "string",
+			"description": "Comma separated zone id. Used im deal targeting & site section targeting. e.g drama,sport"
+		},
+		"dctr": {
+			"type": "string",
+			"description": "Deals Custom Targeting, pipe separated key-value pairs e.g key1=V1,V2,V3|key2=v1|key3=v3,v5"
+		},
```
Comment (AlexBVolcy) [inline static/bidder-params/groupm.json:22]: I noticed what Mansi commented in your docs PR https://github.com/prebid/prebid.github.io/pull/3399, about a difference in bidder params in this file vs. what you have in the docs PR. I was wondering if intended to add any variables from the docs PR to this file, or if you'll just update the docs PR to mirror what's in this groupm.json?

Comment (mansinahar) [reply static/bidder-params/groupm.json:22]: @pm-isha-bharti based on your comment from the docs PR, I am assuming that you don't need to add any  parameters here, correct?

Comment (mansinahar) [reply static/bidder-params/groupm.json:22]: @pm-isha-bharti Just a friendly reminder about the above. If no changes are needed in the bidder params in Prebid Server, I believe we should be able to merge this one

Comment (pm-isha-bharti) [reply static/bidder-params/groupm.json:22]: @mansinahar Yes, we don't need any changes in the bidder params JSON file. I have updated the PR prebid/prebid.github.io#3399 to include PBS bidder params in the documentation

**Review by AlexBVolcy — APPROVED** [review]

**Pull request merged**